modules/node.js

```diff
@@ -170,6 +171,7 @@ Node.prototype.shared = {
 				broadhash: modules.system.getBroadhash(),
 				consensus: modules.peers.getLastConsensus() || null,
 				currentTime: Date.now(),
```

Comment:
So, currentTime can be used to sync a clock, epochTime to estimate transaction timestamp.